doc/user-guide/cli.md

```diff
@@ -334,4 +334,18 @@ To view all labels added to your project simply run:
 
 ```sh 
 ploomber-cloud labels
+```
+
+## Switching the configuration file
+
+By default, the CLI reads and writes to a `ploomber-cloud.json` file, but you can customize it via the `--config` (or its short version, `-c`), switching the config file is useful when you need to manage multiple environments (for example, development and production).
```

Comment:
We can also mention that if there multiple custom config files as well as `ploomber-cloud.json`, running `pc deploy` without specifying the config parameter will default to the `ploomber-cloud.json` file.

Reply:
I think that's implied in the first sentence right? it says that it'll use ploomber-cloud.json by default